Jenkinsfile.demo

```diff
@@ -33,7 +33,7 @@ node {
 
     // Test Docker image
     stage 'Test'
-    sh "docker run -d --name=test-container-${env.BUILD_NUMBER} mesosphere/cd-demo-app:${gitCommit()} jekyll serve"
+    sh "docker run -d --name=test-container mesosphere/cd-demo-app:${gitCommit()} jekyll serve"
```

Comment:
The demo job will no longer be able to run more than one build at a time.

Reply:
Previously the build would only run once, and then fail every other time on that build agent

Reply:
By removing the build number I can clean up the job easily at the end of the pipeline